src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -391,7 +423,7 @@ public static String setInitialValueForVariableDeclaration(
   // simple form of that name
   @SuppressWarnings("signature")
   public static @ClassGetSimpleName String toSimpleName(@DotSeparatedIdentifiers String className) {
-    String[] elements = className.split(".");
+    String[] elements = className.split("[.]");
```

Comment:
Why do we need this change? I think these two regexes are equivalent (but I'm probably not understanding something - I have never been good at regexes).

Reply:
Professor, when I was testing the codes, `split(".")` did not seem to work as expected. I have searched on the Internet, and apparently a dot is a special character. Here is the [link](https://javarevisited.blogspot.com/2016/02/2-ways-to-split-string-with-dot-in-java-using-regular-expression.html#axzz88VNiHntK) that I used.